source/administration/telemetry.rst

```diff
@@ -46,12 +46,15 @@ Mattermost error and diagnostic data is collected for the following purposes:
 .. note:: 
 Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. To opt out, you can disable the feature in **System Console > Environment > Logging** (or **System Console > General > Logging > Enable Error and Diagnostics Reporting** in versions prior to 5.12).
 
-
-The following data is sent once every 24 hours:
+Deployment and Server Configuration snapshots
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Reporting Frequency
+  - When booting up the server for the first time: Every 10 minutes for the first hour, then every hour for the first 12 hours.
```

Comment:
```suggestion
  - When starting the server for the first time: Every 10 minutes for the first hour, then every hour for the first 12 hours.
```

Reply:
"booting up" translates to "start-up" in Japanese - so suggesting the use of "starting" for simplicity

Reply:
ok